patches/tModLoader/Terraria/ModLoader/Engine/LoggingHooks.cs

```diff
@@ -16,7 +17,7 @@ internal static class LoggingHooks
 		}
 
 		private static void HookProcessStart() {
-			new Hook(typeof(Process).GetMethod("Start", BindingFlags.Public | BindingFlags.Instance), new Func<Func<Process, bool>, Process, bool>((orig, self) => {
+			_ = new Hook(typeof(Process).GetMethod("Start", BindingFlags.Public | BindingFlags.Instance), new Func<Func<Process, bool>, Process, bool>((orig, self) => {
```

Comment:
Nice way to silence that warning :). In upcoming new monomod version we're going to have to save them in static vars anyway.